clients/src/main/java/org/apache/kafka/common/log/remote/storage/RLMMWithTopicStorage.java

```diff
@@ -283,7 +312,7 @@ private void initConsumerThread() {
         CommittedLogMetadataFile(File metadataStoreFile) {
```

Comment:
As I discussed offline, if it does not store the state then RLMM may not be available till it reads all the messages fro the remote log metadata topic partitions since the beginning. We are going to add config to maintain local state file or not. If the state file is corrupted then it will read the messages from the topic since the beginning.